src/NuGetForUnity/Editor/PackagesConfigFile.cs

```diff
@@ -212,6 +215,59 @@ public void Save(string filePath)
             contentIsSameAsInFilePath = filePath;
         }
 
+        /// <summary>
+        ///     Moves the packages.config file and its corresponding meta file to the given path. If there is no packages.config file on the current
+        ///     path, makes a default version of the file on the new path.
+        /// </summary>
+        /// <param name="newPath">Path to move packages.config file to.</param>
+        internal static void Move(string newPath)
+        {
+            var oldFilePath = NugetHelper.NugetConfigFile.PackagesConfigFilePath;
+            var oldPath = NugetHelper.NugetConfigFile.PackagesConfigDirectoryPath;
+            NugetHelper.NugetConfigFile.PackagesConfigDirectoryPath = newPath;
+            try
+            {
+                if (!File.Exists(oldFilePath))
+                {
+                    Debug.LogFormat("No packages.config file found. Creating default at {0}", newPath);
+                    var configFile = new PackagesConfigFile { Packages = new List<PackageConfig>() };
+                    configFile.Save();
+                    AssetDatabase.Refresh();
+                    return;
+                }
+
+                var newFilePath = Path.GetFullPath(Path.Combine(newPath, FileName));
+
+                Directory.CreateDirectory(newPath);
+
+                // moving config to the new path
+                File.Move(oldFilePath, newFilePath);
```

Comment:
```suggestion
                File.Move(oldFilePath, newFilePath);
                
            }
            catch (Exception e)
            {
                Debug.LogException(e);
                NugetHelper.NugetConfigFile.PackagesConfigDirectoryPath = oldPath;
            }
```
Resetting to `oldPath` only works if we didn't move the file successfully. I think this is not a problem when we ignore (not explicitly handle) errors occurring later as moving the .meta etc. can be considered "utility" as it is not essential for moving the file.